.github/workflows/publish_precompiled.yml

```
@@ -0,0 +1,70 @@
+name: Publish Precompiled
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  ELIXIR_OPTS: "--warnings-as-errors"
+  ERLC_OPTS: "warnings_as_errors"
+  LANG: C.UTF-8
+  S3_BUCKET: s3.hex.pm
+
+jobs:
+  publish_pre_built:
+    strategy:
+      fail-fast: true
+      # Prevent each job edit builds.txt concurrently.
+      max-parallel: 1
+      matrix:
+        include:
+          - otp: 24
+            otp_version: 24.3
+          - otp: 25
+            otp_version: 25.0
+    runs-on: ubuntu-18.04
```

18.04 is deprecated:

```suggestion
    runs-on: ubuntu-20.04
```

---

Fixed